src/components/templates/TalkShowStudio/components/Player/Player.tsx

```diff
@@ -95,7 +96,7 @@ const Player: React.FC<VideoPlayerProps> = ({
             </div>
 
             {isSharing && (
-              <div className={IconContainerClasses(true)}>
+              <div className={iconContainerClasses(true)}>
```

Comment:
We shouldn't be passing through a raw `true` like this. Either add the correct class here directly (preferred IMO), or pass through the variable.